platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -0,0 +1,485 @@
+events:
+
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable events.
+  #
+  enabled: false
+
+  ## @param: publisher_type
+  ## @supported_values: `kafka`, `sqs`
+  ## The type of queue to use for event publishing
+  ##
+  #
+  publisher_type: kafka
+
+
+  options:
+
+    ## @param: bootstrap_server
+    ## @type: string
+    ## The Kafka server used to bootstrap setup
+    ## For MSK, use port 9098 for access from within AWS and port 9198 for public access and specify
+    ## AWS credentials.
+    ## https://docs.aws.amazon.com/msk/latest/developerguide/port-info.html
+    #
+    bootstrap_server: localhost:29092
+
+    ## @param: use_IAM
+    ## @type: boolean
+    ## Use IAM authentication for AWS MSK or AWS SQS.
+    ## SQS FIFO queues should be pre-created in AWS (Anchor Platform will not create them)
+    ## If true, @required_secrets:
+    ##    AWS_ACCESS_KEY_ID
+    ##    AWS_SECRET_ACCESS_KEY
+    #
+    use_IAM: false
+
+    ## @param: region
+    ## @type: string
+    ## AWS region for the queue. Can also be defined as environment variable: AWS_REGION
+    #
+    aws_region: us-east-1
+
+    ## @param: use_single_queue
+    ## @type: boolean
+    ## If true, all events are published to a single queue (specified in eventTypeToQueue.all)
+    use_single_queue: false
+
+    ## @param: event_type_to_queue
+    ## @type: map
+    ## Mapping of the event type to the queue name that messages are published to
+    event_type_to_queue:
+      all: ap_event_single_queue
+      quote_created: ap_event_quote_created
+      transaction_created: ap_event_transaction_created
+      transaction_status_changed: ap_event_transaction_status_changed
+      transaction_error: ap_event_transaction_error
+
+
+#########################
+## Assets Configuration
+#########################
+## Assets are empty by default. Please see <INSERT_ASSETS_LINK> for reference.
+## Accepts file reference (eg. 'file:assets.yaml') or in-line definition.
+assets:
+  ## @param: type
+  ## @supported_values: `json`
+  ## Describe the asset definition.
+  #
+  type: json
+
+  ## @param: value
+  ## @type: string
+  ## Example of yaml definition:
+  ##   { "assets":
+  ##     [{
+  ##       "schema": "stellar"
+  ##       "code": "USDC"
+  ##       "issuer": "G..."
+  ##       "distribution_account": "G..."
+  ##        ....
+  ##     }]
+  ##   }
+  #
+  value: ""
+
+
+################################
+## Data Configuration
+################################
+data:
+
+  ## DB credentials are specified in @environment_variables DATA_USERNAME, DATA_PASSWORD
+
+  ## @param: type
+  ## @supported_values:
+  ##    `h2` (in-memory), `sqlite` (local), `postgres` (local), `aurora` (postgres on AWS)
+  ## Type of storage.
+  ## If this is set to `aurora`,
+  ##    @required_secrets: AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION
+  #
+  type: h2
+
+  ## @param: url
+  ## @type: string
+  ## Location of the database
+  #
+  url: jdbc:h2:mem:anchor-platform
```

Comment:
Can I specify a normal [database connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) here? why do I need to specify `type`?

Reply:
The type is not limited to RDBMS compatible databases. We may encounter NoSQL.